packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -66,8 +64,8 @@ abstract class ScrollActivity {
   ScrollActivity(this._delegate);
 
   /// The delegate that this activity will use to actuate the scroll view.
-  ScrollActivityDelegate get delegate => _delegate;
-  ScrollActivityDelegate _delegate;
+  ScrollActivityDelegate? get delegate => _delegate;
+  ScrollActivityDelegate? _delegate;
```

Comment:
this should be non-nullable; you can remove the body of the `dispose` method.

Reply:
Done